cpu/cc2538/periph/spi.c

```diff
@@ -72,16 +72,16 @@ void spi_init_pins(spi_t bus)
     switch ((uintptr_t)spi_config[bus].dev) {
         case (uintptr_t)SSI0:
             IOC_PXX_SEL[spi_config[bus].mosi_pin] = SSI0_TXD;
-            IOC_PXX_SEL[spi_config[bus].sck_pin ] = SSI0_CLKOUT;
-            IOC_PXX_SEL[spi_config[bus].cs_pin  ] = SSI0_FSSOUT;
```

Comment:
Maybe this should be handled in a seperate PR, but in other threads we had a discussion concerning hardware chip select. Maybe this would be a good chance to just get rid of the hardware chip select code in the cc2538 spi.

Reply:
Here I just make limited adaptions to the periph/drivers, so they still work with the changed gpio implementation. Changes to the periph/drivers should be discussed (and implemented) in the corresponding PRs, for SPI its #7313.